udata_hydra/utils/json.py

```diff
@@ -1,11 +0,0 @@
-import json
-
-
-def is_json_file(filename: str) -> bool:
-    """Checks if a string is a valid json"""
-    try:
-        with open(filename) as f:
-            json.load(f)
-            return True
-    except ValueError:
-        return False
```

Comment:
Was that leftover code? Well spotted!
Do we have some kind of tooling in the python world to automatically detect leftover code?
Being a dynamic language, I'm not even sure this is possible to be honest 😬